src/admin-orderable-list/messages_en.json

```diff
@@ -0,0 +1,13 @@
+{
+    "adminOrderableList.products": "Products",
+    "adminOrderableList.code": "Code",
+    "adminOrderableList.name": "Name",
+    "adminOrderableList.description": "Description",
+    "adminOrderableList.program": "Program",
+    "adminOrderableList.search": "Search",
+    "adminOrderableList.products.error": "No products found",
+    "adminOrderableList.actions": "Actions",
+    "adminOrderableList.edit": "Edit",
+    "adminOrderableList.addProduct": "Add Product",
+    "adminOrderableList.print": "Print"
+}
```

Comment:
There is no need to copy the whole messages_en.json file, you can just add"adminOrderableList.print": "Print"

Reply:
ok, I will change this